lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -5528,6 +5528,11 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
           if (term->isFunctionExiting()) {
             require(state.Stack.empty(),
                     "return with stack allocs that haven't been deallocated");
+            if (!state.ActiveOps.empty()) {
+              for (auto op : state.ActiveOps) {
+                op->dump();
```

Comment:
Should this be removed?

Reply:
yeah, though i'll admit the reality is adding it back every single time one is debugging sill issues like this... not in love with the pretty bad diagnostics that just "there was a bug" :p Might have another loop to do a pass or improving those diagnostics separately 🤔 

cleanup in https://github.com/apple/swift/pull/41155/